colossalai/fx/__init__.py

```diff
@@ -1,3 +1,10 @@
+try:
+    from . import _meta_registrations
+    META_COMPATIBILITY = True
+except:
+    import torch
+    META_COMPATIBILITY = False
+    print(f'_meta_registrations seems to be incompatible with PyTorch {torch.__version__}.')

```

Comment:
This would still print a statement to the user upon importing colossalai if `colossalai.fx` is used by any other module in the library.

Reply:
Let's not print something like this. Instead, we can offer an API like  `xxx_is_compatible` to check the compatibility.

Reply:
Do not expose a constant to the user, no matter he is a user of the colossalai or the developer who wants to use `colossalai.fx`.

Reply:
I see. For the compatibility concerns, probably I have to come up with a global registration for this. Currently in this PR I will just define an API for checking though.

Reply:
But is there any existing modules in ColossalAI for compatibility checks?

Reply:
this should be only within this fx module as compatibility is specific to this module.